apps/condo/domains/billing/access/BillingReceipt.js

```diff
@@ -59,5 +110,6 @@ module.exports = {
     canReadBillingReceipts,
     canManageBillingReceipts,
     canReadSensitiveBillingReceiptData,
+    getResidentReceiptAccessQuery,
```

Comment:
you should not export anything not related to access functions. You can put helper's utils